capabilities/runtime.md

```diff
@@ -0,0 +1,20 @@
+---
+displayName: Serverless Runtime
+epicsLabels:
+  - area/serverless
+---
+<!-- above metadata will be used on kyma.project.io page to display nice name of capability and have a reference to label that should be used while fetching from ZenHub/GitHub the information about related Epics and their delivery plan   -->
+
+## Scope
+
+The Serverless Runtime capability offering is a central part of the serverless strategy of kyma. It is the easiest way to to run custom code in kyma and to integrate different services provided by the services broker and application connector. Based on the Batteries included strategy of Kyma a FaaS Solution based on Knative is provided inside the OSS Project. Beside the included FaaS Solution it is possible to schedule Workloads on 3rd Party Offerings using knative.
+
+## Vision
+
+The goal of Serverless Runtime capability is to:
+
+- Provide a simple FaaS runtime as part of kyma.
+- Integrate 3rd Party Serverless providers using knative.
+- Integrate FaaS (kyma FaaS and 3rd Party FaaS) with other services provided by Kyma (Eventing, API, Service Catalog)
+- Provide a easy way to run Containerized workloads inside and outside of kyme (using knative)
+- Provide a easy way to host static file for frontend workloads
```

Comment:
I would like to have:
- Make testing and prototyping easy with minimal possible feedback loop length. It can involve local development or hot reload on the cluster - whatever is required to see the result from my changed code in seconds not minutes.
- Enable easy deployment from the version control system (at least have some helpers that can take function.js + package.json and will create yaml out of that and deploy it to cluster, ideally label deployment it with some commit id)

Reply:
Added the two points to the list of goals. As soon as we can go forward with knative they should not be a big deal, at least as we had some of them any way on the list. So fingers crossed after 1.0 we can give full steam.